measure_extinction/tests/test_rebin.py

```diff
@@ -0,0 +1,78 @@
+import numpy as np
+import astropy.units as u
+
+from measure_extinction.stardata import SpecData
+from measure_extinction.extdata import ExtData
+from measure_extinction.merge_obsspec import _wavegrid
+
+
+def test_spec_rebin_constres():
+    spec = SpecData("TEST")
+
+    # fake some simple data
+    # flux values alternate between 1 and 2
+    tres = 1000
+    twaves, tmp, tmp = _wavegrid(tres, [1.001, 10.0])
```

Comment:
Just curious why you are starting from 1.001 and not 1.

Reply:
Was playing around with the rebin code.  Changed it to 1.00 as it doesn't matter and will avoid us asking this question in the future.  ;-)

Reply:
Above explanation is wrong.  This is so that number of wavelength points is even.  Needed to setup the fluxes so that they alternate between values so that the rebinning can be checked for something where the rebinning actually changes the fluxes.  Added an explanation in the code in both tests.

Reply:
Ok, I thought the division with // was making sure a and b are equal length, even if the number of points is uneven.